### ISSUE TITLE
RemovedInDjango19Warning

### ISSUE THREAD

**Issue by mattrobenolt** [omab/python-social-auth#551]
When running in Django 1.8:
```
/Users/matt/.virtualenvs/authy/lib/python2.7/site-packages/social/apps/django_app/default/models.py:29: RemovedInDjango19Warning: Model class social.apps.django_app.default.models.UserSocialAuth doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.
  class UserSocialAuth(models.Model, DjangoUserMixin):

/Users/matt/.virtualenvs/authy/lib/python2.7/site-packages/social/apps/django_app/default/models.py:67: RemovedInDjango19Warning: Model class social.apps.django_app.default.models.Nonce doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.
  class Nonce(models.Model, DjangoNonceMixin):

/Users/matt/.virtualenvs/authy/lib/python2.7/site-packages/social/apps/django_app/default/models.py:78: RemovedInDjango19Warning: Model class social.apps.django_app.default.models.Association doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.
  class Association(models.Model, DjangoAssociationMixin):

/Users/matt/.virtualenvs/authy/lib/python2.7/site-packages/social/apps/django_app/default/models.py:91: RemovedInDjango19Warning: Model class social.apps.django_app.default.models.Code doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.
  class Code(models.Model, DjangoCodeMixin):

/Users/matt/.virtualenvs/authy/lib/python2.7/site-packages/social/apps/django_app/default/models.py:29: RemovedInDjango19Warning: Model class social.apps.django_app.default.models.UserSocialAuth doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.
  class UserSocialAuth(models.Model, DjangoUserMixin):

/Users/matt/.virtualenvs/authy/lib/python2.7/site-packages/social/apps/django_app/default/models.py:67: RemovedInDjango19Warning: Model class social.apps.django_app.default.models.Nonce doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.
  class Nonce(models.Model, DjangoNonceMixin):

/Users/matt/.virtualenvs/authy/lib/python2.7/site-packages/social/apps/django_app/default/models.py:78: RemovedInDjango19Warning: Model class social.apps.django_app.default.models.Association doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.
  class Association(models.Model, DjangoAssociationMixin):

/Users/matt/.virtualenvs/authy/lib/python2.7/site-packages/social/apps/django_app/default/models.py:91: RemovedInDjango19Warning: Model class social.apps.django_app.default.models.Code doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.
  class Code(models.Model, DjangoCodeMixin):
```

**Comment by mandx** [comment]
This is actually useful right now, if one tries to refer to some model using `django.apps.apps.get_model()`. If the `app_label` attribute of the AppConfig instance is not defined, it is created using the last component of the `name` attribute, which in this case, is turns out to be the string `'default'`. So if we need to refer to one of this app's model, we need to call `get_model('default', 'UserSocialAuth')`, which gives no clear indication of which of the installed apps is this `"default"` referring to.

**Comment by wj1918** [comment]
same here, how to fix it?

**Comment by omab** [comment]
Fixed by #550.
